release_guidelines.md

```diff
@@ -16,6 +16,7 @@ An example `<version>` could be `2.0.0-rc.1`
 * Build the dist files with `npm run build`
 * Commit the above using the version as the commit message
 * Create a pull request from the release branch into master
+* Update the JSFiddle demo
```

Comment:
could you convert that into a link to `Update JSFiddle Demo` section for easier reference?